### DISCUSSION TITLE
UNO Platform - Build an Todo App - Part 1

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#5] [General]
# UNO Platform - Build an Todo App - Part 1

This is the beginning of a mini series shedding some lights on the UNO Platform.
The UNO Platform enables us to write code once and run it on a wide range of platforms (Windows, MacOS, Android, Linux, ...).
So let's have a look at this. To do so we will create a small Todo App Kanban-style so we can organize our thoughts and work ;)

https://steven-giesel.com/blogPost/b2234ada-0978-4c7b-841e-ca6a255247b0

**Comment by thiago-tota** [comment]
Very good comparison and step by step examples.
Thanks for sharing.

**Reply by linkdotnet** [reply]
@thiago-tota thanks so much for your support
